src/app/clusters/thread-border-router-management-server/thread-border-router-management-server.cpp

```diff
@@ -168,7 +171,8 @@ void ServerInstance::InvokeCommand(HandlerContext & ctxt)
         break;
     case Commands::SetPendingDatasetRequest::Id:
         HandleCommand<Commands::SetPendingDatasetRequest::DecodableType>(ctxt, [this](HandlerContext & ctx, const auto & req) {
-            ctx.mCommandHandler.AddStatus(ctx.mRequestPath, HandleSetPendingDatasetRequest(req));
+            Status status = HandleSetPendingDatasetRequest(IsCommandOverCASESession(ctx), req);
```

Comment:
Nit: I think a nicer pattern is to just pass both the `HandlerContext` and the request to the `Handle...Request` method instead of adding additional logic to this lambda.

Reply:
Done